src/main/kotlin/com/autonomousapps/Flags.kt

```diff
@@ -16,10 +16,17 @@ object Flags {
   private const val FLAG_MAX_CACHE_SIZE = "dependency.analysis.cache.max"
   private const val FLAG_TEST_ANALYSIS = "dependency.analysis.test.analysis"
   private const val FLAG_PRINT_BUILD_HEALTH = "dependency.analysis.print.build.health"
+  /**
+   * Android build variant to not analyze i.e.
+   *
+   * dependency.analysis.android.ignored.variants=release
+   */
+  const val FLAG_ANDROID_IGNORED_VARIANTS = "dependency.analysis.android.ignored.variants"
```

Comment:
public for reusing in tests

Reply:
It can actually be `private` (which I would prefer) because Groovy can access the private field regardless (Groovy is a jerk).